app/javascript/components/automated-workflows/summary.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Loading, Tabs, Tab, CodeSnippet,
+} from 'carbon-components-react';
+import AWSSfnGraph from '@tshepomgaga/aws-sfn-graph';
+import MiqStructuredList from '../miq-structured-list';
+import { automatedWorkflowData } from './automated-workflows-dummy-data';
+import '@tshepomgaga/aws-sfn-graph/index.css';
+
+const AutomatedWorkflowSummary = ({ recordId }) => {
+  const tabLabels = [
+    { name: 'json', text: 'JSON Data' },
+    { name: 'graph', text: 'Graph' },
+  ];
+  const [data, setData] = useState({
+    isLoading: true,
+    summary: [],
+    jsonData: {},
+    activeTab: tabLabels[0].name,
+  });
+
+  const onClickHandler = (data) => console.log('data=', data);
+
+  // TODO: Change the url when the GET automated worflow summary for recordId api is available.
+  useEffect(() => {
+    API.get('/api/groups?expand=resources')
+      .then((_response) => {
+        const { summary, jsonData } = automatedWorkflowData(recordId);
+        setData({
+          isLoading: false,
+          summary,
+          jsonData,
+        });
+      });
+  }, [recordId]);
+
+  if (data.isLoading) {
+    return (
+      <div className="loadingSpinner">
+        <Loading active small withOverlay={false} className="loading" />
+      </div>
+    );
+  }
+
+  /** Function to set the active tab name */
+  const onTabSelect = (name) => {
+    setData({
+      ...data,
+      activeTab: name,
+    });
+  };
+
+  /** Function to render the code snipper component */
+  const renderCodeSnippet = () => {
+    console.log(data.jsonData);
+    return (
+      <CodeSnippet type="multi">
```

Comment:
I think we have a way to present code already that uses CodeMirror (e.g. what we use for the Ruby code in Automate and the YAML in Advanced Settings).  We should reuse that component, and also use code syntax coloring for JSON.